### PR TITLE
Add get feature method

### PR DESCRIPTION
**Pull request by hadalin**

### CODE REVIEW

File: src/api/types/modules.ts [hunk 1]
```diff
@@ -16,12 +16,17 @@ export interface Feature {
     aliases: string[];
 }
 
-export interface FeatureQuery extends Query {
+export interface FeatureQuery {
+    feature_id: string;
+    source: string;
+}
+
+export interface FeatureSearchQuery extends Query {
```
Comment (hadalin) [inline src/api/types/modules.ts]: Breaking change.

File: src/api/resources/modules/knowledge_base.ts [hunk 2]
```diff
@@ -28,11 +43,36 @@ export class FeatureResource extends KnowledgeBaseResource {
...
+     * @throws `MultipleFeaturesFoundError` is thrown if multiple features found
+     * @throws `NoFeatureFoundError` is thrown if feature not found
+     */
+    public getFeature(query: types.FeatureQuery): Rx.Observable<types.Feature> {
+        const path = this.getModuleMethodPath('search');
+
+        return this.connection.get<types.Feature[]>(path, query).map((features) => {
+            if (features.length > 1) {
+                throw new MultipleFeaturesFoundError(`
+                    Multiple features identified by feature id ${query.feature_id} and source ${query.source}
```
Comment (kostko) [inline src/api/resources/modules/knowledge_base.ts]: This introduces leading and trailing spaces, you should move the quotes.

Comment (hadalin) [inline src/api/resources/modules/knowledge_base.ts]: How about moving backticks in the same line?

**Comment by hadalin** [comment]
Added https://github.com/genialis/resolwe-js/pull/38/commits/c3ec062d5964ea609b524e3cc3873939b07aef5d

**Pull request merged**